tools/build_arrow.sh

```diff
@@ -165,14 +170,50 @@ echo $CURRENT_DIR
 
 cd ${CURRENT_DIR}
 
+ARROW_REPO=https://github.com/oap-project/arrow.git
+
+if [ $BACKEND_TYPE == "velox" ]; then
+    ARROW_BRANCH=backend_velox_main
+elif [ $BACKEND_TYPE == "gazelle" ]; then
```

Comment:
the name is gazellel_cpp

Reply:
Thanks.

Seems to me we have adopted inconsistent naming since the new module was named `backends-gazelle`. Are some further nit changes required here?

Reply:
It use the spark config backend.lib, maybe we should unify that